spec/system/support_interface/application_choices_performance_spec.rb

```diff
@@ -23,7 +29,28 @@ def when_i_visit_the_service_performance_data_page
     visit support_interface_performance_data_path
   end
 
-  def then_i_should_be_able_to_download_a_csv
-    click_link 'Download candidate journey tracking (CSV)'
+  def and_i_click_the_generate_report_link
+    Sidekiq::Worker.clear_all
+    click_link 'Generate candidate journey tracking export'
+  end
+
+  def then_i_see_that_the_export_has_started
+    expect(page).to have_content 'This export is being generated'
+  end
+
+  def when_the_sidekiq_worker_has_finished
+    Sidekiq::Worker.drain_all
```

Comment:
:+1: not seen this way of testing with Sidekiq before.

Reply:
Took me a while to figure out how to do it!